app/src/main/kotlin/band/effective/headlines/compose/di/AppComponent.kt

```diff
@@ -5,10 +5,11 @@ import band.effective.headlines.compose.HeadlinesComposeApp
 import band.effective.headlines.compose.MainActivity
 import band.effective.headlines.compose.about.di.AboutDependencies
 import band.effective.headlines.compose.article_details.di.ArticleDetailsDependencies
-import band.effective.headlines.compose.core.di.CommonDependencies
+import band.effective.headlines.compose.core.di.*
```

Comment:
https://stackoverflow.com/questions/49870306/disable-wild-cart-import-in-intellij-android-studio-in-kotlin